base/docs/basedocs.jl

```diff
@@ -1300,61 +1300,61 @@ isdefined
 
 
 """
-    Vector{T}(n)
+    Vector{T}(uninitialized, n)
 
 Construct an uninitialized [`Vector{T}`](@ref) of length `n`.
```

Comment:
I guess we can now `@ref` uninitialized here?

Reply:
Meh, a bit weird maybe.

Reply:
Good call! Linking to `uninitialized`, either from that word in the existing sentence or via a `See also...` addendum, is a great idea. Which do you prefer? Thanks!

Reply:
Perhaps a ``See [`uninitialized`](@ref).``